vk-video/src/parser/nalu_parser.rs

```diff
@@ -144,6 +145,14 @@ pub enum ParsedNalu {
     Other(String),
 }
 
+/// H264 Network Abstraction Layer Unit
+pub struct Nalu {
+    /// Parsed nalu from [`Nalu::raw_bytes`]
+    pub parsed: ParsedNalu,
+    pub raw_bytes: Bytes,
```

Comment:
I'm not a huge fan of putting `Bytes` in the public API, and I would like to stay with a `Vec<u8>`. What do you think about it? Can you measure the performance impact of one vs the other (e.g. how big chunk of a profile NAL-splitting is with a `Vec` here vs with `Bytes`)? If the performance benefit is significant I think it's OK to keep `Bytes` here, but intuitively I don't think there is much difference.